policies.go

```diff
@@ -175,10 +183,18 @@ func (s *SimpleRetryPolicy) Attempt(q RetryableQuery) bool {
 	return q.Attempts() <= s.NumRetries
 }
 
+func (s *SimpleRetryPolicy) AttemptLWT(q RetryableQuery) bool {
+	return s.Attempt(q)
+}
+
 func (s *SimpleRetryPolicy) GetRetryType(err error) RetryType {
 	return RetryNextHost
 }
 
+func (s *SimpleRetryPolicy) GetRetryTypeLWT(err error) RetryType {
```

Comment:
Could you please add comments on why LWT retry should end up on the same host.

Reply:
Done